doc/api/videoThumbnailLoader.md

```diff
@@ -0,0 +1,130 @@
+# VideoThumbnailLoader #########################################################
+
+
+## Overview ####################################################################
+
+The VideoThumbnailLoader is a tool that can help exploiting a trickmode video
+track to provide thumbnails for a video content.
+
+The goal is to make a thumbnail out of HTML5 video element, by :
+- Managing the loading / appending of resources from a given track
+(video segments).
+- Exploiting the Media Source Extension API to make it invisible to user.
+
+The tool will need the loaded manifest to contain trickmode tracks. These kind
+of track exists in MPEG-DASH and HLS, and contains lightweight video tracks,
+most of the time including one unique frame for each video segments. As video
+segments from trickmode tracks may be quicker to load and easier to decode, they
+are preferred over standard video tracks for creating thumbnails.
+
+## How to use it ###############################################################
+
+
+As an experimental tool, the VideoThumbnailLoader won't be included in a
+default RxPlayer build.
+
+Instead, it should be imported by adding the RxPlayer through a dependency
+trough the npm registry (e.g. by doing something like ``npm install
+rx-player``) and then specifically importing this tool from
+``"rx-player/experimental/tools"``:
+
+```js
+import { VideoThumbnailLoader } from "rx-player/experimental/tools";
+import RxPlayer from "rx-player";
+
+const player = new RxPlayer({ /* some options */ });
+player.loadVideo({ /* some other options */ });
+
+// Video element used to display thumbnails.
+const thumbnailVideoElement = document.createElement("video");
+const videoThumbnailLoader = new VideoThumbnailLoader(
+  thumbnailVideoElement,
+  player
+);
```

Comment:
Would be nice to also add a LOADER here (probably DASH) for a full working example